packs/rcon_web/README.md

```diff
@@ -0,0 +1,108 @@
+|------|-------------|------|
+| app_rwa_admin | Toggle to set Initial User as Admin. | `bool` |
+| app_rwa_env | Toggle to allow configuration through Environment Variables. | `bool` |
+| app_rwa_game | Initial Game-Type. | `string` |
+| app_rwa_password | Initial User Password. | `string` |
+| app_rwa_rcon_host | RCON Target Server Host. | `string` |
+| app_rwa_rcon_password | RCON Target Server Password. | `string` |
+| app_rwa_rcon_port | RCON Target Server Port. | `number` |
+| app_rwa_read_only_widget_options | Toggle to prevent Initial User from changing Widget Options. | `bool` |
+| app_rwa_restrict_commands | Restricted Commands for Initial User. | `string` |
+| app_rwa_restrict_widgets | Hidden Widgets for Initial User. | `string` |
+| app_rwa_server_name | Name of Target Server. | `string` |
+| app_rwa_username | Initial User Username. | `string` |
+| app_rwa_web_rcon | Toggle to enable Web RCON on Target Server. | `bool` |
+| count | Count of Deployments for the Job. | `number` |
+| datacenters | Eligible Datacenters for the Task. | `list(string)` |
+| driver | Driver to use for the Job. | `string` |
+| ephemeral_disk | Ephemeral Disk Configuration for the Application. | <pre>object({<br>    migrate = bool<br>    size    = number<br>    sticky  = bool<br>  })</pre> |
+| group_name | Name for the Group. | `string` |
+| image | Content Address to use for the Container Image. | <pre>object({<br>    registry  = string<br>    namespace = string<br>    image     = string<br>    tag       = string<br>    digest    = string<br>  })</pre> |
+| job_name | Name for the Job. | `string` |
+| job_tags | List of Tags for the Job. | `list(string)` |
+| namespace | Namespace for the Job. | `string` |
+| network_mode | Network Mode for the Job. | `string` |
+| ports | Port Configuration for the Application. | <pre>map(object({<br>    name           = string<br>    path           = string<br>    port           = number<br>    type           = string<br>    host_network   = string<br>    check_interval = string<br>    check_timeout  = string<br>  }))</pre> |
+| priority | Priority for the Job. | `number` |
+| region | Region for the Job. | `string` |
+| resources | Resource Limits for the Application. | <pre>object({<br>    cpu        = number<br>    cores      = number<br>    memory     = number<br>    memory_max = number<br>  })</pre> |
+| restart_logic | Restart Logic for the Application. | <pre>object({<br>    attempts = number<br>    interval = string<br>    delay    = string<br>    mode     = string<br>  })</pre> |
+| service_name_prefix | Name for the Service. | `string` |
+| service_provider | Provider for the Service. | `string` |
+| task_name | Name for the Task. | `string` |
+| verbose_output | Toggle to enable verbose output. | `bool` |
+| volumes | Volumes for the Application. | <pre>map(object({<br>    name        = string<br>    type        = string<br>    destination = string<br>    read_only   = bool<br>  }))</pre> |
```

Comment:
Great call!

Since this is automatically rendered, I'll not accept this suggestion (as it would be overwritten, next time).

I've added an issue [here](https://github.com/workloads/nomad-pack-registry/issues/5) to track this, to fix it in a better way.